src/test/scala/nl/knaw/dans/lib/logging/servlet/MapExtensionsSpec.scala

```diff
@@ -30,6 +42,28 @@ class MapExtensionsSpec extends FlatSpec with Matchers {
     input.makeString shouldBe "[a -> foo, b -> bar, c -> baz, d -> qux]"
   }
 
+  it should "take special care of values that are a Seq with no elements (empty square brackets)" in {
+    val input = Map(
+      "a" -> "".toSeq,
+      "b" -> "".toSeq,
+      "c" -> "".toSeq,
+      "d" -> "".toSeq,
+    )
+
+    input.makeString shouldBe "[a -> [], b -> [], c -> [], d -> []]"
```

Comment:
Like the dropped quotes I don't see the trouble in reading: `"[a ->, b ->, c ->, d ->]"`. Hmmm, except that you can't tell the difference between null or an empty sequence.